packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -137,7 +137,10 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
     this.printOnNewline("import Apollo");
   }
 
-  classDeclarationForOperation(operation: Operation) {
+  classDeclarationForOperation(
+    operation: Operation,
+    outputIndividualFiles: boolean
```

Comment:
I'd add a docstring to describe what exactly this parameter is doing (maybe with examples of the difference in output?), because it's not very clear to me just looking at it.